src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -251,22 +300,20 @@ def _guessSmallestFov(self, ss):
         return (min(f[0] for f in fovs),
                 min(f[1] for f in fovs))
 
-    def _getNumberOfTiles(self):
+    def _getTileCoverage(self):
```

Comment:
Please, use the `snake_case` style. Also while at it, add type annotation (about the return values).